docs/installation/upgrade/include/2.4-added-cassandra-parameters.inc

```diff
@@ -0,0 +1,7 @@
+#### Cassandra Parameters
+
+See [[link,sql_cassandra]].
+
+| Parameter | Notes |
+| --------- | ----- |
+| `log_retries` | Parameter was added. |
```

Comment:
This is v3.0 only. v2.4 has cassandra_log_retries setting. Except looks like it's actually missing, need to add it.

Reply:
but it's not in 2.3 either?

Reply:
Right. So this is another example of how should we separate v2.4 vs v3.0 vs v3.1 changes. If you add `cassandra_log_retries` here it's correct for v2.4 and v3.1, but then incorrect if you include it in v3.0.

Reply:
Oh, this is still `log_retries`. It needs to be `cassandra_log_retries`.